README.md

```diff
@@ -11,20 +11,37 @@ We decided to opensource this tool as we believe that it will help people to ado
 
 ## Installation
 
-It uses Go Modules, so GO 1.11+ is required.
-If you wish to run it via docker then `docker` binary is required.
-While if you wish to run it locally; you will need to have the extra `brz` lib installed.
+### Binary
 
-You have 2 options to get the package:
+Visit the [releases](https://github.com/cycloidio/terracognita/releases) page to select your system, architecture and version you need. Then:
 
-* Clone the repository to `$GOPATH/src/github.com/cycloidio/terracognita`
-* `go get -d github.com/cycloidio/terracognita`
+```shell
+$ curl -L https://github.com/cycloidio/terracognita/releases/download/v0.3.0/terracognita-linux-amd64.tar.gz -o terracognita-linux-amd64.tar.gz
+$ tar -xf terracognita-linux-amd64.tar.gz
+$ chmod u+x terracognita-linux-amd64
+$ sudo mv terracognita-linux-amd64 /usr/local/bin/terracognita
+```
+
+### Development
 
-Then feel free to play with it :)
+You can build and install with the latest sources, you will enjoy the new features and bugfixes. It uses Go Modules, so GO 1.11+ is required.
```

Comment:
Mention also that `brz` is required

Reply:
Just to keep a log: 
>  an alpine image with `git ca-certificates go` were enough to go get !

`brz` seems to not be needed anymore !